modules/core/tests/client/header.client.controller.tests.js

```diff
@@ -16,48 +16,45 @@
       $state = _$state_;
       Authentication = _Authentication_;
 
-      HeaderController = $controller('HeaderController', {
+      HeaderController = $controller('HeaderController as vm', {
```

Comment:
Oh wait, these are in the tests, sorry I just noticed. Then this looks fine to me and same with the home tests. Only thing is the extra IIFE above that I think can be cut out.